package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vscode-xml",
   "displayName": "XML",
   "description": "XML Language Support by Red Hat",
-  "version": "0.13.0",
+  "version": "0.14.0",
```

Comment:
Thanks for this change. Could we have it in a separate commit, either in this PR or a separate PR?

Reply:
Will add it in a separate PR